clang/test/3C/definedType.c

```diff
@@ -109,7 +109,7 @@ baz (*(*lua_test3));
 
 typedef int *StkId;
 void lua_test4(StkId *x) {}
-// CHECK: void lua_test4(_Ptr<_Ptr<int>> x) _Checked {}
+//CHECK: void lua_test4(_Ptr<StkId> x) _Checked {}
```

Comment:
Is this happening because `StkId` is used unsafely at some other point in this test? The old test wouldn't have cared about that, since it drops the typdefs, I imagine.